lldb/source/ValueObject/DILEval.cpp

```diff
@@ -21,6 +21,42 @@
 
 namespace lldb_private::dil {
 
+lldb::ValueObjectSP
+GetDynamicOrSyntheticValue(lldb::ValueObjectSP in_valobj_sp,
+                           lldb::DynamicValueType use_dynamic,
+                           bool use_synthetic) {
+  Status error;
+  if (!in_valobj_sp) {
+    error = Status("invalid value object");
```

Comment:
is this even used?
Should the function return `llvm::Expected<ValueObjectSP>` instead?